instrumentation/net/http/otelhttp/common.go

```diff
@@ -31,10 +31,17 @@ const (
 
 // Server HTTP metrics.
 const (
-	RequestCount          = "http.server.request_count"           // Incoming request count total
-	RequestContentLength  = "http.server.request_content_length"  // Incoming request bytes total
-	ResponseContentLength = "http.server.response_content_length" // Incoming response bytes total
-	ServerLatency         = "http.server.duration"                // Incoming end to end duration, milliseconds
+	ServerRequestCount          = "http.server.request_count"           // Incoming request count total
```

Comment:
Could we update the `otelhttp` package documentation to include the information about the emited metrics? Something like https://pkg.go.dev/go.opentelemetry.io/contrib/instrumentation/runtime

This can be addressed as a separate issue and PR.

Reply:
I will defer that to a separate issue/PR

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/issues/4774